publish-aspnet-selfcontained/test.json

```diff
@@ -5,6 +5,9 @@
   "version": "8.0",
   "versionSpecific": false,
   "type": "bash",
-  "cleanup": true
+  "cleanup": true,
+  "skipWhen": [
+    "portable", // skip during cross-bld-sdk/tarball validation
```

Comment:
```suggestion
    "portable", // portable doesn't know about platform RIDs
```

Reply:
It would be better to run this test (by using the SDK rid) than to skip it.

Reply:
That's a good idea!

The current test runs `./runtime-id` to get the non-portable RID. Is there some way to identify/use the SDK's RID?

Reply:
`dotnet --info` prints it under `RID`. You can also get it from the MSBuild `NETCoreSdkRuntimeIdentifier` property.